### ISSUE TITLE
After upgrading to Nexus IPTV simplex client restart the system

### ISSUE THREAD

**Issue by martusi61** [kodi-pvr/pvr.iptvsimple#777]
After upgrading from CoreElec Matrix 19.2 to CoreElec Nexus 20.2, IPTV simple client restart the system, I tried insert any m3u list, but the problem still persist. I also deleted the configuration files in userdata and uninstalled and reinstalled the addon without success, it starts, but as soon as I insert any list, restart the tvbox
My system:
TV box Tanix TX3 mini
CoreElec Nexus 20.2
IPTV simple client 20.10.1.1

**Comment by lrusak** [comment]
It's sounds like Kodi is crashing. Can you post a debug log (and crash log)?

**Comment by martusi61** [comment]
> Sembra che Kodi stia andando in crash. Puoi pubblicare un registro di debug (e un registro degli arresti anomali)?

[kodi_crash.log](https://github.com/kodi-pvr/pvr.iptvsimple/files/12102121/kodi_crash.log)

**Comment by phunkyfish** [comment]
From your log I can see It loads the channels just fine and there is no EPG file to load.

I’d say you have a corrupt DB file or something.

Can you do a completely fresh install on your device, then only install iptvsimple, configure it and see if it works?

**Comment by martusi61** [comment]
Yes, the loading starts, but when it reaches 50% it stops for a few seconds and then restarts the tvbox. In order to use Kodi, I have to delete the IPTV configuration every time, between reboots. A clean installation on that device is very difficult for me, but tell me those DB are you referring to, I could try to delete it or reload it from an old backup.

**Comment by phunkyfish** [comment]
In userdata/database I think. There should be tv and epg databases. Maybe even a PVR one.

Should just be able to delete them and they will be auto recreated.

**Comment by martusi61** [comment]
Under userdata database I found files starting with TV and EPG, so I deleted them. So I restarted my tvbox and IPTV started working perfectly. I am very grateful to you for the help you gave me after so many tests that I had conducted without success. You pointed me towards the solution. I know others have had the same problem as me and I hope they read this solution.

**Comment by phunkyfish** [comment]
No problem at all. Happy to help.

**Comment by casedup** [comment]
How or where were you able to install the plugin? when I updated to Nexus 20.0 iptv simple client is disabled due to compatibility issues.

**Comment by phunkyfish** [comment]
> How or where were you able to install the plugin? when I updated to Nexus 20.0 iptv simple client is disabled due to compatibility issues.

It should auto update your plugins assuming you have auto update enabled.

**Comment by casedup** [comment]
Yup I have it enabled... The problem also is PVR Client is not even a option in the Kodi Official Repo to choose an App to download. Very weird I had no choice but to go back to Matrix. I think the problem might lie in the fact im using LibreElec

**Comment by phunkyfish** [comment]
Auto update on LibreElec should also work. That’s the platform I use myself and I had no issues. In fact I’m not sure it’s possible to use old add-one on LibreElec as they ship with the installer.

**Comment by phunkyfish** [comment]
Did you try to manually update the add-on in
The add-on menu?

**Comment by ilayasayag** [comment]
Has anyone solved this issue yet?
I'm trying to find a solution to this problem for a long time but haven't succeeded yet.

**Comment by casedup** [comment]
@phunkyfish says it should auto update with the LibreElec upgrade, maybe there is some step that I am missing. The problem with auto updating though is that there is no PVR Client section inside of the official Kodi Repo for Nexus.. I thought maybe it was a LibreElec bug or something but you said you are also using the same OS and it worked perfectly.

**Comment by ilayasayag** [comment]
I'm using Kodi Nexus on my Android TV and I don't think I even have LibreElec on my device.

IPTV Simple Client Crashes my Kodi every time and Although I think previous versions of Kodi would work, I want to find a stable solution for the future.

Also, If anyone has alternative M3U8 player that includes Catch Up I would like to hear

**Comment by phunkyfish** [comment]
> @phunkyfish says it should auto update with the LibreElec upgrade, maybe there is some step that I am missing. The problem with auto updating though is that there is no PVR Client section inside of the official Kodi Repo for Nexus.. I thought maybe it was a LibreElec bug or something but you said you are also using the same OS and it worked perfectly.

I’m pretty sure you need to select the LibreElec repo for the binary addons. The kodi repo is for Python addons only.

**Comment by phunkyfish** [comment]
> I'm using Kodi Nexus on my Android TV and I don't think I even have LibreElec on my device.
> 
> IPTV Simple Client Crashes my Kodi every time and Although I think previous versions of Kodi would work, I want to find a stable solution for the future.
> 
> Also, If anyone has alternative M3U8 player that includes Catch Up I would like to hear

Please open a separate issue and provide a full debug log and I can take a look.
